resources/services/lambda/functions.go

```diff
@@ -1423,6 +1423,10 @@ func resolveFunctionEventSourceMappingsSourceAccessConfigurations(ctx context.Co
 	return diag.WrapError(resource.Set(c.Name, data))
 }
 
-// ====================================================================================================================
-//                                                  User Defined Helpers
-// ====================================================================================================================
+func resolveFunctionsTags(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	r := resource.Item.(*lambda.GetFunctionOutput)
+	if r.Tags == nil {
+		return resource.Set(c.Name, make(map[string]string))
+	}
+	return resource.Set(c.Name, r.Tags)
+}
```

Comment:
Is there a reason why we don't use the `client.TagsToMap` or `client.TagsIntoMap`?